.github/workflows/ff.yml

```diff
@@ -0,0 +1,32 @@
+name: Fast-Forward PR
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  fast_forward_job:
+    name: Fast Forward
+    if: >
+        github.event.issue.pull_request != '' &&
+        contains(github.event.comment.body, '/fast-forward') &&
+        ${{ github.actor == 'hsaunders1904' || github.actor == 'je-cook' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code into workspace directory
+        if: ${{ steps.comment-branch.outputs.head_ref == 'develop_dependencies' }}
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      # Basic use case example
+      - name: Fast Forward PR
+        if: ${{ steps.comment-branch.outputs.head_ref == 'develop_dependencies' }}
+        id: ff-action
+        uses: endre-spotlab/fast-forward-js-action@master
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          success_message: 'Success! Fast forwarded ***target_base*** to ***source_head***! ```git checkout target_base && git merge source_head --ff-only``` '
+          failure_message: 'Failed! Cannot do fast forward!'
+          staging_branch: "develop_dependencies"
+          production_branch: "develop"
```

Comment:
Does this mean it will merge `develop_dependencies` into `develop` no matter which PR we comment on? It's not clear from the action's docs.

Reply:
Done!